llumnix/queue/ray_queue_server.py

```diff
@@ -22,6 +22,8 @@
 from llumnix.utils import random_uuid
 
 
+# TODO(KuilongCui): make rayqueueserver and api server stay in the same node
+# for local launch mode
```

Comment:
not only local launch mode, but also global launch mode, i think this is high priority work item